src/main/java/Duke.java

```diff
@@ -11,21 +14,29 @@ public static void main(String[] args) {
         System.out.println("-------------------------------");
         System.out.println("Hello! I'm Duke\n" +
                 "What can I do for you?");
-
-
         System.out.println("-------------------------------");
 
         String echo = " ";
+        ArrayList<String> addedList = new ArrayList<>();
 
         Scanner input = new Scanner(System.in);
         while (!echo.equals("bye")) {
             echo = input.nextLine();
-            if(!echo.equals("bye")){
+            if(!echo.equals("bye") && !echo.equals("list")){
+                addedList.add(echo);
                 System.out.println("-------------------------------");
                 System.out.println("    " + echo);
                 System.out.println("-------------------------------");
             }
-
+            if(echo.equals("list")){
+                int index = 0;
+                System.out.println("-------------------------------");
+                Iterator itr = addedList.iterator();
+                while (itr.hasNext()){
+                    System.out.println("    " + ++index +". " + itr.next());
```

Comment:
I like that the loop body is wrapped by curly brackets irrespective of how many lines there are in the body.